GliaWidgets/Sources/Component/AttachmentList/AttachmentSourceListStyle.swift

```diff
@@ -59,3 +59,20 @@ extension AttachmentSourceListStyle {
         lhs.backgroundColor == rhs.backgroundColor
     }
 }
+
```

Comment:
please wrap it with `#if DEBUG`

Reply:
fixed [f3566d1](https://github.com/salemove/ios-sdk-widgets/pull/845/commits/f3566d1688549ebaf323e9078b956ebe650b0270)